common/src/View/FaceAttribsEditor.cpp

```diff
@@ -77,6 +77,7 @@ namespace TrenchBroom {
             createGui(contextManager);
             bindEvents();
             bindObservers();
+            gridDidChange();
```

Comment:
I'd rename the function `updateIncrements`, it feels strange to call `gridDidChange` from the constructor. If you decided not to rename, I'd leave a comment here because it's unusual.